docs/cluster-management/configure-artifacts-unzip-service.md

```diff
@@ -74,6 +74,7 @@ WORKER_MIN_TASK_PROCESSORS | 1 | The minimum number of workers to spawn under th
 WORKER_MAX_TASK_PROCESSORS | 10 | The maximum number of workers to spawn under this multiWorker.
 WORKER_CHECK_TIMEOUT | 1000 | How often to check if the event loop is blocked (in ms)
 WORKER_MAX_EVENT_LOOP_DELAY | 10 | How long the event loop has to be delayed before considering it blocked (in ms)
+WORKER_PARALLEL_UPLOAD_LIMIT| 0 | Number of parallel executions of decompressing ZIP files and uploading to the Store (unlimited when 0)
```

Comment:
I think technically we do not decompress Zip file. How about just  "Number of parallel executions of uploading to the Store"

Also "unlimited when less than 0" is more precisely than "unlimited when 0".
https://github.com/screwdriver-cd/artifacts-unzip-service/blob/main/lib/jobs.js#L55

Reply:
thx.
I fixed it.